app/models/opentofu_worker.rb

```diff
@@ -5,6 +5,8 @@ class OpentofuWorker < MiqWorker
   self.rails_worker          = false
   self.maximum_workers_count = 1
 
+  STARTING_PORT = 6000
```

Comment:
I think this is only used by our rails workers, right?

Reply:
This is only used by `MiqWebServerWorkerMixin` so yeah we probably don't need this